apps/mobile/src/features/Discover/common/ledger.ts

```diff
@@ -449,7 +449,7 @@ export async function toLedgerSignRequest(
   }
 
   const txBody = csl.FixedTransaction.fromHex(cbor).body()
-  const parsedCbor = decode(txBody.toBytes())
+  const parsedCbor = decode<Map<number, unknown>>(txBody.toBytes())
```

Comment:
@lisicky can confirm 
```suggestion
  const parsedCbor = decode<Map<number, UInt8Array>>(txBody.toBytes())
```

Reply:
Not so sure, it's called `parsedCbor` so I think it's no longer binary, and it uses:
```
parsedCbor.get(1)?.constructor?.name === 'Map'
```
So I don't think it's a UInt8Array. In any case, `unknown` is enough here for ts not to complain, and it's not really propagated anywhere else.

Reply:
@stackchain @jorbuedo Instead using cbor2’s decode you can just use `serialization_format` function from transaction output.